modules/check_in/app/services/v2/chip/client.rb

```diff
@@ -98,14 +99,24 @@ def pre_check_in(token:, demographic_confirmations:)
       end
 
       ##
-      # HTTP POST call to the CHIP API to set pre check-in started status
+      # HTTP POST call to the CHIP API to set pre check-in started status. Any downstream error (non HTTP 200 response)
+      # is handled by logging to Sentry and returning the original status and body.
       #
       # @return [Faraday::Response]
       #
       def set_precheckin_started(token:)
         connection.post("/#{base_path}/actions/set-precheckin-started/#{check_in_session.uuid}") do |req|
           req.headers = default_headers.merge('Authorization' => "Bearer #{token}")
         end
+      rescue => e
```

Comment:
It is possible but I don't think we should do that. The goal of this is to allow the Veteran's to continue pre checking in if the **set status** call fails. If CHIP's token call fails, that means we don't have a valid session with CHIP and is indicative of a larger problem and we shouldn't be masking it, and it should show an error on the website. 

Moreover, if we do allow them to continue pre check-in without being able to get a valid CHIP token, it will still fail after they confirm pre check-in because that call will try to get a token and fail (which it should).

Reply:
Just noticing, we are not handling the response in the controller. But handling the exception in the service layer would prevent missing the notification in sentry in case of issues with the token as we are not calling pre-check-in started endpoint if token is not present.

Reply:
I'm not sure I understand your comment. Not sure what "not handling the response in the controller" has to do with it. But my main point is that it doesn't make sense to allow users to continue pre check-in if there are any issues getting the token, as that will and should cause blocking failures.

Reply:
As synced offline, this is a bit of a bigger change and we will move forward with the current implementation of handling exceptions in the client layer.